docs/UserGuide.md

```diff
@@ -503,16 +503,18 @@ All students in class CS2103T has been marked present!
 
 ### Starting a lesson : `startlesson /c`
 
-Starts a lesson of your chosen class. This increases the total number of lessons conducted in your class and automatically marks all students as absent for attendance taking.
+Starts a lesson of your chosen class and facilitates the update of various class records for you. 
```

Comment:
Just my opinion but i think the phrase "facilitates the update of various class records for you" sounds a bit general. Perhaps you can change to something like: "Starts a lesson of your chosen class and now you are ready to take attendance."

Reply:
Okay thanks!